mos-platform/common/c/stdio-full.c

```diff
@@ -8,12 +8,13 @@
 #include <errno.h>
 #include <fcntl.h>
 #include <limits.h>
-#include <stdbool.h>
```

Comment:
`stdbool.h` is coming in https://github.com/llvm-mos/llvm-mos-sdk/pull/357.

Reply:
We do already have `<stdbool.h>`; it just comes from clang, not the SDK.